tests/e2e/machine_pool_test.go

```diff
@@ -651,6 +651,64 @@ var _ = Describe("TF Test", func() {
 			})
 		})
 	})
+
+	Describe("Validate terraform Import operations", func() {
+		var mpService *exe.MachinePoolService
+		var importService exe.ImportService
+
+		BeforeEach(func() {
+			mpService = exe.NewMachinePoolService(con.MachinePoolDir)
+			importService = *exe.NewImportService(con.ImportResourceDir) // init new import service
```

Comment:
How about import service destroy in AfterEach? What will happen when destroy an imported resource?

Reply:
I fixed